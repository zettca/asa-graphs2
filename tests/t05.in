30 50
17
2 13 -3
27 9 0
25 28 -3
30 21 3
17 18 -3
27 15 -2
7 19 -1
23 14 2
14 8 0
29 13 1
13 27 -2
2 14 2
10 9 -3
16 23 3
5 14 0
26 1 -1
18 11 2
3 12 -1
21 2 1
8 2 -1
3 28 0
5 30 -1
30 6 -1
18 7 3
1 11 0
18 10 -2
30 9 -3
20 16 3
27 20 0
18 29 -2
14 2 2
22 4 0
3 6 -1
12 24 -1
18 21 -2
13 21 -3
3 1 -3
16 22 1
14 10 -3
4 5 3
11 25 -3
2 30 -2
15 26 0
5 26 2
2 25 3
10 3 1
26 14 1
7 8 0
19 17 1
6 22 -2
