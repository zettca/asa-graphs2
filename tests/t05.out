I
I
I
I
I
I
0
I
I
I
I
I
I
I
I
I
0
-3
-1
I
I
I
I
I
I
I
I
I
-5
I
